src/os-dependent.h

```diff
@@ -52,86 +42,9 @@ inline unsigned long long GetTickCount64(void)
 
 #define PATH_SEPARATOR_CHAR '/'
 
-#if defined(__APPLE__)
-// for HRESULT
-#include <CoreFoundation/CFPlugInCOM.h>
-#endif
-
-/* Handling of 2-byte Windows wchar strings on non-Windows targets
- * Used by The MediaPortal and ForTheRecord pvr addons
- */
-typedef uint16_t Wchar_t; /* sizeof(wchar_t) = 4 bytes on Linux, but the MediaPortal buffer files have 2-byte wchars */
-
-/* This is a replacement of the Windows wcslen() function which assumes that
- * wchar_t is a 2-byte character.
- * It is used for processing Windows wchar strings
- */
-inline size_t WcsLen(const Wchar_t *str)
-{
-  const unsigned short *eos = (const unsigned short*)str;
-  while( *eos++ ) ;
-  return( (size_t)(eos - (const unsigned short*)str) -1);
-};
-
-/* This is a replacement of the Windows wcstombs() function which assumes that
- * wchar_t is a 2-byte character.
- * It is used for processing Windows wchar strings
- */
-inline size_t WcsToMbs(char *s, const Wchar_t *w, size_t n)
```

Comment:
Please keep this! Is is essential since a Windows wchar_t != Linux wchar_t
The file I'm reading is in Windows format (2-bytes) and a Linux wchar_t is 4 bytes.

Reply:
Will do